persistence/scheduled-tasks/schedule-task-via-command-line.yml

```diff
@@ -6,14 +6,22 @@ rule:
       - 0x534a@mailbox.org
     scope: function
     att&ck:
+      - Persistence::Scheduled Task/Job::At [T1053.002]
       - Persistence::Scheduled Task/Job::Scheduled Task [T1053.005]
```

Comment:
i wonder if we should split these rules out so that the techniques don't get mixed (e.g. don't want to tag "AT job" if at wasn't used).